examples/gifify.py

```diff
@@ -0,0 +1,12 @@
+from PIL import Image
```

Comment:
Since we said in Readme that we don't include plotting requirement, this is probably okay. But is this package in the `fv3viz` environment?

Reply:
No it's not, it's more a small utility to make gif from the command line